.claude/scripts/generate-pr-table.sh

```
@@ -0,0 +1,121 @@
+sanitize_pr_text() {
+    local input="$1"
+    
+    # 制御文字・リダイレクト記号を除去
+    echo "$input" | \
+        sed 's/< \/dev\/null//g' | \
+        sed 's/<\/dev\/null//g' | \
+        sed 's/< *\/dev\/null *//g' | \
+        sed 's/>>>.*<<<//g' | \
+        sed 's/<<<.*>>>//g' | \
+        tr -d '\000-\010\013\014\016-\037\177' | \
+        sed 's/\$([^)]*)//g' | \
```

Shellcheck (0.10.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)